packages/datagateway-dataview/src/views/landing/isis/isisBranding.component.tsx

```diff
@@ -22,6 +22,17 @@ const useStyles = makeStyles((theme: Theme) =>
     },
     typography: {
       color: theme.palette.primary.contrastText,
+      '& a': {
+        '&:link': {
+          color: '#FF6900',
```

Comment:
Can't we get the colour directly from the theme object that is passed to us via SciGateway? We would need to ignore typescript for accessing unknown properties perhaps but it would mean not having to change the colour here if we change the colour in SciGateway

Reply:
I wasn't sure how to reference this in all honesty. It's easy in SciGateway:
`(theme as UKRITheme)`

but `UKRITheme` doesn't exist in DataGateway. What's the best method of doing this?

Reply:
It's a choice between either duplicating the colour code or perhaps ts-ignoring this line and accessing the property via optional chaining. Thoughts? I think I prefer the latter method.